src/schema/v2/homeView/sections/DiscoverSomethingNew.ts

```diff
@@ -0,0 +1,46 @@
+import { ContextModule } from "@artsy/cohesion"
+import { HomeViewSection } from "."
+import { HomeViewSectionTypeNames } from "../HomeViewSection"
+import { connectionFromArray } from "graphql-relay"
+
+const marketingCollectionSlugs = [
+  "figurative-art",
+  "new-from-leading-galleries",
+  "paintings",
+  "prints",
+  "street-art",
+  "black-and-white",
+  "art-under-1000",
+  "art-for-small-spaces",
+  "cool-toned-artworks",
+]
+
+export const DiscoverSomethingNew: HomeViewSection = {
+  id: "home-view-section-discover-something-new",
+  featureFlag: "diamond_home-view-marketing-collection-categories",
+  contextModule: "" as ContextModule, // TODO: fill in with real value
+  type: HomeViewSectionTypeNames.HomeViewSectionCards,
+  component: {
+    title: "Discover Something New",
+  },
+  requiresAuthentication: false,
+  resolver: async (_parent, args, context, _info) => {
```

Comment:
**suggestion (blocking)**: We're keeping this data static for now because we're still figuring out the final content and whether the curation team will manage it. I’d recommend keeping the data as is for now and introducing the network call later.

Reply:
Do we have many examples of when the desired display text differs from the text we already have on the backend?

Looking at [the static data](https://github.com/artsy/metaphysics/blob/0416dcd2f1a2309bb318f90d546bd13a23f9ebd2/src/schema/v2/homeView/sections/DiscoverMarketingCollections.ts#L23-L79), it looks like we're often mirroring the same values that already exist and are easily editable on the backend.

Reply:
My concern is more related to the endpoiont we are using, as we're currently requesting marketing data. The previous thought was focused on `marketing_categories` with potential API changes.

Reply:
This section instance ("Discover Something New") seems to be very MarketingCollection-specific, so it seems okay to use the marketingCollectionsLoader and fetch some of these values from the backend.

For categories, I agree, it's best to stick with static data as we don't yet have first class modeling or endpoints/loaders to use.

Does match your understanding as well? Or is there another reason to use the MarketingCollection loader here specifically?

Reply:
That makes sense!

Reply:
Yeah I see that two of the display titles here differ from the collections’ current titles:

- **Figurative Art** instead of **Figuration**
- **Black-and-White Artworks** instead of **Black-and-White** 

We could re-title those (though there may be consistency considerations, with respect to how other similar collections are named — I think Jenna/Addy would be best to confirm that)

We could also add a `display_title` field to the `MarketingCollection` model if this seems like a common enough need. (`Gene` models have something similar)